examples/ELZ-ReferralComposition-01.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Composition xmlns="http://hl7.org/fhir">
+    <id value="ELZ-ReferralComposition-01"/>
+    <status value="final"/>
+    <type>
+        <coding>
+            <system value="http://loinc.org"/>
+            <code value="57133-1"/>
+            <display value="Referral note"/>
+        </coding>
+    </type>
+    <subject>
+        <reference value="Patient/ELZ-ReferralServiceRequest-01-Patient-01"/>
+        <type value="Patient"/>
+        <display value="Marijne van XXX_Prins"/>
+    </subject>
+    <date value="2022-10-05"/>
+    <author>
+        <reference value="PractitionerRole/ELZ-ReferralServiceRequest-01-PractitionerRole-01"/>
+        <type value="PractitionerRole"/>
+        <display value="dr. B.A. van Galen"/>
+    </author>
+    <title value="Referral note"/>
+    <section>
```

Comment:
Is there a reason we currently don't have examples that include the sections carePath and requiredConsultationFacilities?

Reply:
We'll add this later to our examples.